### DISCUSSION TITLE
clock on only 1 monitor i have 2

### DISCUSSION THREAD

**Discussion by keitht1958** [martinet101/ElevenClock#1318] [Q&A]
before reinstall of win 11 i had eleven clock on 2 monitors black background and green font which  i liked
now since reinstall i have black font no coloured background and on 1 monitor only
windows is updated and i have latest Nvidia driver

**Comment by marticliment** [comment]
Do you have NVidia GeForce Overlay running?

**Comment by keitht1958** [comment]
Yes I do I have now turned it off

**Reply by marticliment** [reply]
Does it work now?

**Comment by keitht1958** [comment]
Still the same restarted pc no change just on 1 monitor

**Comment by keitht1958** [comment]
I installed   <https://github.com/marticliment/ElevenClock/releases/tag/4.3.0> ElevenClock 4.3.0 and now it ia on on both monitors and just how I like it thx
